docs/source/develop.rst

```diff
@@ -7,31 +7,45 @@ and documentation and style standards are available at the `Dask developer
 guidelines`_ in the main documentation.
 
 .. _Dask: http://dask.org
-.. _`Dask developer guidelines`: http://docs.dask.org/en/latest/develop.html
 
 Install
 -------
 
-After setting up an environment as described in the `Dask developer
-guidelines`_ you can clone this repository with git::
+Clone this repository with git::
 
    git clone git@github.com:dask/distributed.git
+   cd distributed
 
-and install it from source::
+Install all dependencies to match CI:
 
-   cd distributed
-   python setup.py install
+On Linux / MacOSX::
+
+    source continuous_integration/travis/install.sh
+
+On Windows:
 
-Using conda, for example::
+1. Install anaconda or miniconda
+2. ::
+
+    conda create -n distributed python=3.8 tornado=6
+    conda activate distributed
+    conda env update --file continuous_integration/environment-windows.yml
+    python -m pip install .
```

Comment:
Why the difference between linux/mac and windows here?  Is there not also a script for windows?

Reply:
As a matter of fact, there isn't. The "script" for Windows is ``.github/workflows/ci-windows.yaml``. To make it clean I should extract these lines from it and write a .BAT file, but I don't have a Windows box readily available and I'm not comfortable with DOS scripting. So I chose to be conservative here.